static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-ab-thrasher.js

```diff
@@ -0,0 +1,99 @@
+define([
+    'common/utils/config',
+    'common/utils/detect'
+], function (
+    config,
+    detect
+) {
+    return function () {
+        var self = this;
+
+        this.id = 'MembershipBundlesThrasher';
+        this.start = '2017-01-23';
+        this.expiry = '2017-03-02'; // Thursday 2nd March
+        this.author = 'Justin Pinner';
+        this.description = 'Test appetite for membership bundles';
+        this.showForSensitive = true;
+        this.audience = 0.05;  // 5% ~ 7,000 unique browser thrasher views per variant per day
+        this.audienceOffset = 0;
+        this.successMeasure = '';
+        this.audienceCriteria = 'People on UK network front with at least an 1140px wide display.';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'One landing page will accrue a higher number of intention interactions than the other';
+
+        this.canRun = function () {
+            // call me paranoid but...
+            return document.querySelector('#membership-ab-thrasher') &&
+                detect.isBreakpoint({min: 'leftCol'}) &&
+                config.page.isFront &&
+                config.page.pageId.toLowerCase() === "uk" &&
+                config.page.edition.toUpperCase() === 'UK';
+        };
+
+        this.thrasherContainer = function() {
+            return document.querySelector('#membership-ab-thrasher');
+        };
+
+        this.thrasher = function() {
+            return document.querySelector('.membership-ab-thrasher--wrapper');
+        };
+
+        this.setCopy = function() {
+            if (this.thrasher()) {
+                var mainTitleEl = document.querySelector('.membership-ab-thrasher_header .main_title');
+                if (mainTitleEl) {
+                    mainTitleEl.innerHTML = '<p>Support<br>the Guardian</p>';
+                }
+                var subTitleEl = document.querySelector('.membership-ab-thrasher_header .sub_title');
+                if (subTitleEl) {
+                    subTitleEl.innerHTML = '<p>We\'re introducing <strong>new ways</strong> to support<br>the Guardian\'s quality journalism and independent voice. Choose from digital,<br>print or a contribution today.</p>';
```

Comment:
No, the default body text is there as a backstop in the event of something somewhere making the thrasher visible unexpectedly. The default text is lifted from the current supporter sign up page and the button links through to there, so the experience will still make sense if this happens.